lldb/tools/lldb-dap/package.json

```diff
@@ -28,11 +28,11 @@
   ],
   "devDependencies": {
     "@types/node": "^18.11.18",
-    "@types/vscode": "~1.74.0",
-    "@vscode/vsce": "^2.19.0",
-    "prettier-plugin-curly": "^0.1.3",
-    "prettier": "^3.1.1",
-    "typescript": "^4.6.4"
+    "@types/vscode": "1.75.0",
```

Comment:
Should we use ^1.75.0?

Reply:
The engine field should indicate which version of VS Code is supported, this should only unblock any newer APIs.

Reply:
The engine field states that we support `^1.75.0`. I.e., we also claim support with version 1.75. Now, if I would declare `@types/vscode` as `^1.75`, I might end up with `@types/vscode` version 1.80. This would allow me to use APIs which were only introduced in 1.80, and I would accidentally break compatibility with 1.75, despite our `engine` property claiming that we would be compatible with 1.75.

Reply:
Thanks for explaining!